src/routines/get_results.jl

```diff
@@ -48,7 +48,32 @@ function _result_dataframe_variables(variable::JuMP.Containers.DenseAxisArray)
     end
 
 end
-
+function _result_dataframe_duals(constraint::JuMP.Containers.DenseAxisArray)
+    if length(axes(constraint)) == 1
+        result = Vector{Float64}(undef, length(first(constraint.axes)))
+        for t in constraint.axes[1]
+            try result[t] = JuMP.dual(constraint[t])
+            catch
```

Comment:
Suggest catching only the exact exception you're expecting.  Otherwise, someone might end up with a confusing problem to debug.